create_notional_data.txt

```diff
@@ -0,0 +1,105 @@
+from rackcity.models import ITInstance, ITModel, Rack
+Run `python manage.py shell` and run the following commands
```

Comment:
Couldn't figure out how to make this a script that would run with the right settings, so keeping it as .txt file for now